litespi/modules.py

```diff
@@ -0,0 +1,12300 @@
+# Generated using 'spi_nor_cfg_gen.py'
+
+class AT25FS010(object):
+
+    manufacturer_id = 0x1f
+    device_id = 0x6601
+    name = "at25fs010"
+
+    total_size =     131072   # bytes
+    page_size  =        256   # bytes
+
+    access_modes = {
+        1 : True,
+        2 : False,
+        4 : False,
+        8 : False,
+    }
+    addr_widths = {
+        1 : True,
+        2 : False,
+        4 : False,
+        8 : False,
+    }
+    fast_read_support = True
+    addr32_support = False
```

Comment:
Why is addr32_support and fast_read_support seperate from the access_modes and addr_widths?

Reply:
Fixed it. They are now part of [SpiNorConfig](https://github.com/litex-hub/litespi/pull/1/files?file-filters%5B%5D=.patch&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&hide-deleted-files=true#diff-6f4a7670a48198f06e6250d0ccda1601)

Reply:
**SpiNorConfig** replaced with **SpiNorFeatures**